tests/integration/test_random_advanced.py

```diff
@@ -110,12 +117,13 @@ def test_geometric(t):
 
 
 @pytest.mark.parametrize("t", BITGENERATOR_ARGS, ids=str)
-def test_zipf(t):
+@pytest.mark.parametrize(
+    "s", (7.5, pytest.param((1.2, 3.1415), marks=pytest.mark.xfail)), ids=str
+)
+def test_zipf(t, s):
     bitgen = t(seed=42)
     gen = num.random.Generator(bitgen)
-    s = 7.5
     a = gen.zipf(a=s, size=(1024 * 1024,), dtype=np.uint32)
-    a = np.random.zipf(s, 1024 * 1024)
```

Comment:
I guess this test was just completely broken before? What is the failure mode now?

Reply:
It was. Error comment added to the test.